### PR TITLE
Upgrading page

### PR DESCRIPTION
**Pull request by pornel**

### CODE REVIEW

File: documentation/upgrading/index.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Updates now must either use DSA keys correctly, or not try to use them at all. Same goes for Apple Code Signing.
+
+Sparkle will print to the console verbose messages detailing problems found. Try updating and see Console.app if the updates are rejected.
+
+## Upgrading from Sparkle 1.6 and older
+
+Sparkle has kept the same API since the "classic" version 1.5. It should be a drop-in replacement apart from updated system requirements:
+
+### Mac OS X 10.7+ required
```
Comment (jakepetroules) [inline documentation/upgrading/index.md]: OS X

File: documentation/upgrading/index.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+To avoid sending incompatible app to users on Mac OS 10.6 or older, add to your appcast `<item>`:
```
Comment (jakepetroules) [inline documentation/upgrading/index.md]: OS X

File: documentation/upgrading/index.md (hunk 1 above)
Comment (zorgiepoo) [inline documentation/upgrading/index.md]: To elaborate, it's "OS X" not "Mac OS X" =)

Comment (pornel) [inline documentation/upgrading/index.md]: Well, when 10.7 was released it was still called Mac OS X.

File: documentation/upgrading/index.md (hunk 2 above)
Comment (jakepetroules) [inline documentation/upgrading/index.md]: But 10.6 was not called Mac OS ;)

Comment (pornel) [inline documentation/upgrading/index.md]: I've fixed that one.

**Pull request merged**